CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.16.2
```

Comment:
Note: if you don't plan to release a new version immediately after the PR, let's not fill the version right away, but write "Unreleased" in the header. Who knows, maybe the next release will be a major bump